tox.ini

```diff
@@ -64,24 +76,41 @@ commands =
 
 [testenv:robotdocs]
 changedir = {toxinidir}
-deps=
+deps =
     crl.devutils
-commands=
+commands =
     crl create_robotdocs -v
 
 [testenv:test]
+basepython = python3.7
 changedir = {toxinidir}
-deps=
+deps =
     {[testenv:docs]deps}
 
-commands=
+commands =
     crl test --no-virtualenv {posargs}
 
-[testenv:robottests]
-deps=
+[robotbase]
+deps =
     robotframework
-    crl.remotescript
+    tox
 
-commands=
-    pybot --loglevel TRACE --exclude skip  --exclude skip   {posargs}   \
+commands =
+   robot --loglevel TRACE --exclude skip  --exclude skip   {posargs}   \
           {toxinidir}/robottests
+
+[testenv:robottests2]
+deps = {[robotbase]deps}
+basepython = python2.7
+commands = {[robotbase]commands}
+
+[testenv:robottests3]
+deps = {[robotbase]deps}
+basepython = python3.7
+commands = {[robotbase]commands}
+
+
```

Comment:
Please remove this line

Reply:
Not sure which line. The two empty lines (111-112) or all four lines (109-112)?